### DISCUSSION TITLE
Is it possible to open a pdf in the browser instead of downloading the file

### DISCUSSION THREAD

**Discussion by TerryLiang404** [requarks/wiki#6385] [Help / Questions]
Is it possible to open a pdf in the browser instead of downloading the file?

**Comment by cdn0x12** [comment]
Yes, but not possible through changing the settings at the moment. It could be done by setting the following response headers:
```
Content-Type: application/pdf
Content-Disposition: inline; filename="Filename.pdf"
```

**Comment by ja49619** [comment]
> Yes, but not possible through changing the settings at the moment. It could be done by setting the following response headers:
> 
> ```
> Content-Type: application/pdf
> Content-Disposition: inline; filename="Filename.pdf"
> ```

hi can you explain how to do this? I have wikijs 2.5.302
